include/flexflow/model.h

```diff
@@ -638,9 +638,37 @@ class FFModel {
     return this->new_node(op);
   }
 
+  template <typename T>
+  PCG::Node get_or_create_node(const std::vector<ParallelTensor>& inputs, typename T::Params const& params) {
```

Comment:
What do you think about doing something like this:

```suggestion
  PCG::Node get_or_create_node(typename T::InputType const &inputs, typename T::Params const& params) {
```

Reply:
Good idea. And we need an interface to obtain input shape from input for each operator. Am I true?